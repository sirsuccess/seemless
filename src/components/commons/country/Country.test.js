import React from "react";
import renderer from "react-test-renderer";

import Component from "./Country";

it("renders correctly when called ", () => {
  const tree = renderer.create(<Component />).toJSON();
  expect(tree).toMatchSnapshot();
});
