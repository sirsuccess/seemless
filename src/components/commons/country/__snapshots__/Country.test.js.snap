// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when called  1`] = `
<div
  className="Country"
>
  Nigeria
</div>
`;
